src/terrasoApi/user/userFragments.ts

```diff
@@ -60,9 +44,3 @@ export const userPreferences = `
   }
   ${userPreferencesFields}
 `;
-
-export type UserPreferencesFragment = {
```

Comment:
Question: I see that this type is deleted, but it is still imported in accountService.js, how is that working?

Reply:
it is being generated automatically because of the `/* GraphQL */` comment